sirepo/beaker_compat.py

```diff
@@ -7,7 +7,7 @@
 """
 from __future__ import absolute_import, division, print_function
 
-from beaker import session
+from beaker.session import SignedCookie
```

Comment:
This is opposite of what I wrote. I'm confused.

Reply:
Oops, I put it back the way it was instead of fully qualifying, one sec...